tox.ini

```diff
@@ -4,7 +4,8 @@ envlist = py27,py35,py36
 
 [testenv]
 commands=
-    pip install --global-option=build_ext --global-option=--gdal-config=/usr/bin/gdal-config --global-option=-I/usr/include/gdal GDAL=={env:GDAL_VERSION:2.1.3}
+    pip install --global-option=build_ext --global-option=--gdal-config=/usr/bin/gdal-config --global-option=-I/usr/include/gdal GDAL=={env:GDAL_VERSION:2.3.1}
```

Comment:
@FaiBarazi, can you add this as a separate environment as we still rely on 2.1.3 on xenial w/ ubuntugis so it would be good to test on both of these and maybe 2.2.3 as well as that's the stable version on bionic?